pkg/credentialprovider/keyring.go

```diff
@@ -111,6 +111,11 @@ func isDefaultRegistryMatch(image string) bool {
 		return true
```

Comment:
This code is confusing because the comment gives "library/ubuntu" as an example, but len(strings.SplitN("library/ubuntu", "/", 2)) == 2 (not 1).